VTEX - Orders API.json

```diff
@@ -12591,6 +12382,19 @@
                     }
                 }
             }
+        },
+        "parameters": {
+            "reason": {
+                "name": "reason",
+                "in": "query",
+                "description": "This parameter is relevant only for PII-compliant accounts. When sending requests to this endpoint, PII-compliant accounts can use this parameter to declare the reason for requesting unmasked data. Otherwise this endpoint will return masked PII data.",
```

Comment:
```suggestion
                "description": "This parameter is relevant only for PII-compliant accounts. When sending requests to this endpoint, PII-compliant accounts can use this parameter to declare the reason for requesting unmasked data. Otherwise, this endpoint will return masked PII data.",
```